runtime/libpgmath/lib/CMakeLists.txt

```diff
@@ -235,23 +235,19 @@ endif()
 # Create libmath static library
 get_property(TARGET_OBJECTS GLOBAL PROPERTY "TARGET_OBJECTS")
 
-add_library(${LIBPGMATH_LIBRARY_NAME}_static STATIC ${TARGET_OBJECTS})
-add_library(${LIBPGMATH_LIBRARY_NAME} SHARED ${TARGET_OBJECTS})
-target_link_libraries(${LIBPGMATH_LIBRARY_NAME} INTERFACE ${LIBPGMATH_LIBRARY_NAME}_static)
-if(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME}_static PROPERTIES OUTPUT_NAME lib${LIBPGMATH_LIBRARY_NAME})
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
-if(${CMAKE_GENERATOR} MATCHES "MSYS Makefiles")
-add_custom_command (TARGET ${LIBPGMATH_LIBRARY_NAME} PRE_LINK
-    COMMAND  sed 's/ /\\n/g' CMakeFiles/pgmath.dir/objects1.rsp > objects1.txt
-    COMMAND  ${CMAKE_COMMAND} -E __create_def libpgmath.def objects1.txt
-)
-endif()
+add_library(pgmath_shared SHARED ${TARGET_OBJECTS})
+add_library(pgmath_static STATIC ${TARGET_OBJECTS})
 
-else()
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME}_static PROPERTIES OUTPUT_NAME ${LIBPGMATH_LIBRARY_NAME})
-endif()
-install(TARGETS ${LIBPGMATH_LIBRARY_NAME}
-  LIBRARY DESTINATION lib)
-install(TARGETS ${LIBPGMATH_LIBRARY_NAME}_static
-  ARCHIVE DESTINATION lib)
+target_link_libraries(pgmath_shared INTERFACE pgmath_static)
+set_target_properties(pgmath_shared PROPERTIES OUTPUT_NAME pgmath WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+set_target_properties(pgmath_static PROPERTIES OUTPUT_NAME ${_lib}pgmath)
+
+install(TARGETS pgmath_shared
+        LIBRARY DESTINATION ${LLVM_LIBRARY_DIR}
+        ARCHIVE DESTINATION ${LLVM_LIBRARY_DIR}
+        RUNTIME DESTINATION ${LLVM_TOOLS_BINARY_DIR})
+
+install(TARGETS pgmath_static
+        LIBRARY DESTINATION ${LLVM_LIBRARY_DIR}
+        ARCHIVE DESTINATION ${LLVM_LIBRARY_DIR}
+        RUNTIME DESTINATION ${LLVM_TOOLS_BINARY_DIR})
```

Comment:
```suggestion
install(TARGETS pgmath_shared
        LIBRARY DESTINATION lib
        ARCHIVE DESTINATION lib
        RUNTIME DESTINATION bin)

install(TARGETS pgmath_static
        LIBRARY DESTINATION lib
        ARCHIVE DESTINATION lib
        RUNTIME DESTINATION bin)
```

Reply:
Is this okay?

Reply:
This is fine. I was just wondering how `LLVM_LIBRARY_DIR` and friends got defined for this CMakeLists.txt, because currently we build libpgmath independently of flang. I understand that those variables would be inherited if libpgmath was built as part of flang.

Reply:
In my patched windows version, I don't build libpgmath independently.